### PR TITLE
Exporting saliency classes to Python

### LINKED ISSUE #681: Dose module "saliency" work in python?
It does not seem like the "saliency" has python bindings.

I success build and import cv2 in python.
But "saliency" not work in python, though it work in c++.

This is very useful if it can use.

### PR DESCRIPTION
**Pull request by sovrasov**
resolves #681
replaces #462

### CODE REVIEW

File: modules/saliency/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
 set(the_description "Saliency API")
 set(OPENCV_MODULE_IS_PART_OF_WORLD OFF)
+add_definitions(-DOBJECTNESS_MODELS_PATH="${CMAKE_INSTALL_PREFIX}/${OPENCV_OTHER_INSTALL_PATH}/Objectness")
```
Comment (mshabunin) [inline modules/saliency/CMakeLists.txt]: I don't think it's a good approach.

**Comment by vpisarev** [comment]
thanks! :+1:

**Comment by saeed68gm** [comment]
What this change reverted at the end? so still no saliency for python?

**Comment by sovrasov** [comment]
@saeed68gm saliency works, from CMake was removed a couple of redundant definitions.

**Pull request merged**